packages/react-components/react-skeleton/stories/Skeleton/SkeletonRow.stories.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { Skeleton, SkeletonItem, SkeletonProps } from '@fluentui/react-skeleton';
+import { makeStyles, tokens } from '@fluentui/react-components';
+
+const useStyles = makeStyles({
+  invertedWrapper: {
+    backgroundColor: tokens.colorNeutralBackground1,
+  },
+});
+
+const SkeletonElementsFirstRow = () => {
+  return (
+    <div style={{ display: 'flex', paddingBottom: '10px', position: 'relative' }}>
+      <SkeletonItem shape="circle" size={24} style={{ margin: '5px' }} />
+      <SkeletonItem size={16} style={{ margin: '5px', width: '80%' }} />
+    </div>
+  );
+};
+const SkeletonElementsSecondRow = () => {
+  return (
+    <div style={{ display: 'flex', paddingBottom: '10px', position: 'relative' }}>
+      <SkeletonItem shape="circle" size={24} style={{ margin: '5px' }} />
+      <SkeletonItem size={16} style={{ margin: '5px', width: '20%' }} />
+
+      <SkeletonItem size={16} style={{ margin: '5px', width: '20%' }} />
+      <SkeletonItem size={16} style={{ margin: '5px', width: '15%' }} />
+      <SkeletonItem size={16} style={{ margin: '5px', width: '15%' }} />
+    </div>
+  );
+};
```

Comment:
Our examples should feature best practices, so when consumers will copy them to their projects they will follow them 😉 

I don't think that using `style` everywhere in this example is a recommended way to do things, I suggest to refactor the whole example or its components to use Grid Layout. For example:

```tsx
const useStyles = makeStyles({
  secondRow: {
    alignItems: 'center',
    display: 'grid',
    paddingBottom: '10px',
    position: 'relative',
    ...shorthands.gap('5px'),
    ...shorthands.margin('5px'),
    gridTemplateColumns: 'min-content 20% 20% 15% 15%',
  },
});

const SkeletonElementsSecondRow = () => {
  const classes = useStyles();

  return (
    <div className={classes.secondRow}>
      <SkeletonItem shape="circle" size={24} />
      <SkeletonItem size={16} />
      <SkeletonItem size={16} />
      <SkeletonItem size={16} />
      <SkeletonItem size={16} />
    </div>
  );
};
```

Reply:
I've updated the example to reflect this!